nestjs/src/courses/interviews/interviews.controller.ts

```diff
@@ -96,4 +106,59 @@ export class InterviewsController {
 
     throw new BadRequestException('Invalid interview id');
   }
+
+  // use `type` as a way to differentiate between stage-interview and interview.
+  @Get('/:interviewId/:type/feedback')
+  @ApiOkResponse({ type: InterviewFeedbackDto })
+  @ApiForbiddenResponse()
+  @ApiBadRequestResponse()
+  @ApiOperation({ operationId: 'getInterviewFeedback' })
+  @RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager])
+  public async getInterviewFeedback(
+    @Param('courseId', ParseIntPipe) _: number,
+    @Param('interviewId', ParseIntPipe) interviewId: number,
+    @Param('type') type: 'stage-interview' | 'interview',
+    @Req() req: CurrentRequest,
+  ) {
+    const { user } = req;
+
+    if (type !== 'stage-interview') {
+      throw new BadRequestException('Only stage interviews are supported now.');
+    }
+
+    const interview = await this.interviewFeedbackService.getStageInterviewFeedback(interviewId, user.githubId);
+
+    return new InterviewFeedbackDto(interview);
+  }
+
+  @Post('/:interviewId/:type/feedback')
+  @ApiOkResponse()
+  @ApiForbiddenResponse()
+  @ApiBadRequestResponse()
+  @ApiOperation({ operationId: 'createInterviewFeedback' })
+  @RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager])
```

Comment:
@RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager], true)

Reply:
you are my hero, thanks!